src/core/stream_send.c

```diff
@@ -1436,6 +1474,10 @@ QuicStreamOnAck(
                     "[strm][%p] Send State: %hhu",
                     Stream,
                     QuicStreamSendGetState(Stream));
+                QuicTraceLogStreamVerbose(
+                    GracefulShutdown,
+                    Stream,
+                    "Shutting down [gracefully] from OnStreamAck. Send Side.");
```

Comment:
Is this really needed?

Reply:
I found it useful while debugging with TextAnalysis.